### ISSUE TITLE
Internal links using double square brackets

### ISSUE THREAD

**Issue by holroy** [gsantner/markor#486]
#### General information

* **App version:** v1.6.0 (45)
* **System:** Android 6.0.1

#### Description

I'm collating all of my markdown documents into a small personal wiki, and for this to work I would really love to have internal links easily available. So therefore I'm asking for the addition of internal links using double square brackets.

Some examples.
* `[[Something]]` to link to the file `Something.md`
* `[[WikiLink]]` to link to the file `WikiLink.md`
* `[[MyLongWikiLink|links]` links to the file `MyLongWikiLink.md`, but displays "links"

In other words, I would like for Markor to support internal MediaWiki (piped) links, see https://www.mediawiki.org/wiki/Help:Links#Internal_links .

**Comment by holroy** [comment]
If I'm not mistaken you're already using the flexmark parser, and it has support for this in the _wikilink_ extension, see [flexmark wikilink examples](https://github.com/vsch/flexmark-java/blob/cda02f72d57b5c60d5f71b7dc0fd62bfb5d9f240/flexmark-ext-wikilink/src/test/resources/ext_wikilink_ast_spec.md).

**Comment by gsantner** [comment]
What if I want to refer to `file.txt` with `[[file]]`?

I want Markor to stay mostly compatible with other markdown editors/convertors and I'm not knowing any other using this^^
